packages/action-group/README.md

```diff
@@ -30,55 +30,55 @@ import { ActionGroup } from '@spectrum-web-components/action-group';
 ```html
 <sp-action-group>
     <sp-action-button>
-        <sp-icon slot="icon" size="xs" name="ui:Magnifier"></sp-icon>
+        <sp-icon slot="icon" size="m" name="ui:Magnifier"></sp-icon>
```

Comment:
I've been using "feat" changes for breaking change to ensure nothing accidentally went to 1.0, but I'll confirm that's the actual functionality or not, pre-1.0.

For this, great call, and it is very sad/why I was bringing this up to Spectrum CSS last week. Hopefully t-shirt sizing will reduce or remove the need for this in the future. However, can you help me understand the wording for this breaking change? I say this because it's not a "break" so much as a "migration" in that your app will still "work". I do read this note as wanting the information in the CHANGELOG.md for the package, is it OK if that's the only place it's listed?

Reply:
Rebasing these notes into makes me feel like a wizard, but `BREAKING CHANGE:` causes `1.0`'s to be cast. I can keep the notes and remove `BREAKING CHANGE:` or do something else? Use `MIGRATION:` maybe?

Reply:
Oh hmmm... that's unfortunate if it pushes to 1.0 :(

I was just trying to get us into a better habit...

Reply:
MIGRATION seems better than nothing.